.babelrc

```diff
@@ -15,21 +15,8 @@
     ["@babel/plugin-proposal-object-rest-spread", { useBuiltIns: true }],
     ["module-resolver", {
       "alias": {
-        "firefox-profiler": "./src",
+        "firefox-profiler": "./src"
       }
     }]
-  ],
-  env: {
-    test: {
-      presets: [
-        "@babel/preset-env",
-        "@babel/preset-react",
-        ["@babel/preset-flow", { all: true }]
-      ],
-      plugins: [
-        ["@babel/plugin-proposal-class-properties", { loose: true }],
-        ["@babel/plugin-proposal-object-rest-spread", { useBuiltIns: true }],
-      ],
-    }
-  }
```

Comment:
This was exactly the same as the "normal" one since we removed the special babel transforms for the dynamic imports.